lib/workers/repository/process/lookup/timestamps.ts

```diff
@@ -0,0 +1,33 @@
+import type { ReleaseResult } from '../../../../modules/datasource/types';
+import type { VersioningApi } from '../../../../modules/versioning/types';
+import { type Timestamp, asTimestamp } from '../../../../util/timestamp';
+
+export function calculateLatestReleaseTimestamp(
```

Comment:
For npm do you define "latestReleaseTimestamp" as the timestamp of the release tagged as `latest`? Or do you simply mean "the most recent timestamp of any release"? Because I think to change the name if it's the latter, e.g. to `mostRecentReleaseTimestamp` or `newestReleaseTimestamp`.

Reply:
I would go further and name it `bumpedAt`